### ISSUE TITLE
Generate oneToManyMultiset-methods when there are multiple references between the tables

### ISSUE THREAD

**Issue by alf** [jOOQ/jOOQ#13621]
jOOQ doesn't generate any oneToManyMultiset methods from `Per` to `Tel` in the following schema where we have two references between the tables:

```
CREATE TABLE Per
  ( per_id  int not null
  , per_id2 int not null
  , name varchar2(30)
  )
;

ALTER TABLE Per
  ADD CONSTRAINT Per__PK
      PRIMARY KEY (per_id)
;  

ALTER TABLE Per
  ADD CONSTRAINT Per__UK
      UNIQUE (per_id2)
;  

CREATE TABLE Tel
  ( tel_id int not null
  , per_id int not null
  , per_id2 int not null
  , num varchar2(30)
  )
;

ALTER TABLE Tel
  ADD CONSTRAINT Tel__Per__FK
      FOREIGN KEY    (per_id)
      REFERENCES Per (per_id)
;

ALTER TABLE Tel
  ADD CONSTRAINT Tel__Per__FK2
      FOREIGN KEY    (per_id2)
      REFERENCES Per (per_id2)
;
```

The above case is of course over simplified. It would be nice if jOOQ could generate methods for these situations the same way it generates the `tel_Per_Fk` and `tel_Per_Fk2` methods from `Tel` to `Per`.

This is on the 3.17.0-SNAPSHOT from 20220527202542.

**Comment by lukaseder** [comment]
Thanks a lot for your report. You should see `INFO` level a log entry:

```
Multiple references from child table cannot be disambiguated yet: <foreign key>
```

It's a known issue in https://github.com/jOOQ/jOOQ/issues/13069, see:

> Special case: There are multiple paths between two many-to-many connected tables

It also applies on one-to-many relationships. The problem is that we can't yet generate an "inverse" name from a foreign key relationship using our generator strategies. This should be resolved by the time of release 3.17.

**Comment by lukaseder** [comment]
Duplicate of #13069
